src/librustc_back/svh.rs

```diff
@@ -313,10 +313,10 @@ mod svh_visitor {
     // Ad-hoc overloading between Ident and Name to their intern table lookups.
     trait InternKey { fn get_content(self) -> token::InternedString; }
```

Comment:
@eddyb: I could implement something like this... maybe `From<Ident/Name> for InternedString`?